packages/admin-ui/src/pages/stakers/components/StakerNetwork.tsx

```diff
@@ -233,7 +233,7 @@ export default function StakerNetwork<T extends Network>({
                   isSelected={newEnableWeb3signer}
                 />
               </Col>
-              {(network === "prater" || network === "mainnet") && (
+              {(network === "prater" || network === "mainnet" || network === "holesky") && (
```

Comment:
Probably there is a better way to do this. Something like:
`["prater", "mainnet", "holesky"].includes(network)`

(research a bit on the best way to do it)

Reply:
The use of `Array.includes()` makes the condition more expressive and self-explanatory. It clearly communicates the intent of checking whether the network value is among a predefined list of valid networks. So it is a better approach for readability, maintainability, and flexibility of the code.